packages/ui/src/types/primitives/constants.ts

```diff
@@ -0,0 +1,117 @@
+export enum ComponentClassName {
```

Comment:
Maybe consider naming the file _componentClassName.ts_? Thinking it might be confusing for future readers to find a _constants.ts_ nested inside a _types_ dir

Reply:
Good point! Done in [3c3e5f0](https://github.com/aws-amplify/amplify-ui/pull/2751/commits/3c3e5f059465c40bfac74e7c661726a99003505f).

Reply:
Looking forward, we might need a separate file for `Authenticator` class names. That's something we can strategize and adjust filenames later though, so don't want to worry about it now.

Reply:
If we're going to expose this publicly via ui package, wouldn't we need to do that change now? Once this is exposed we can't change it.